bin/rad-project

```diff
@@ -57,9 +57,20 @@
     (fetch! rsm)
     (def name (lookup :name (get-meta! rsm)))
     (def origin (lookup :id (first-rsm-of-type! rsm :rad-repo)))
-    (put-str! name)
-    (process! "git" ["clone" origin name] "")
-    (cd! name)
+    (match (process! "test" ["-d" name] "")
+      :ok (do
+            (put-str! (string-append
+                        "The directory \"" name "\" already exists. "
+                        "It needs to be removed to checkout the project in this directory."))
+            (exit! 1))
+      _   (put-str! name))
+    (match (process! "git" ["clone" origin name] "")
+      :ok (cd! name)
+      _   (do
+            (put-str! (string-append
+                        "The repo is currently not available. "
+                        "Cloning \"" name "\" from \"" origin "\" failed."))
```

Comment:
Maybe add something like `"This may be because you have no internet connection, or the IPNS link is stale, or because no online node with the data could be found."`